docs/source/usage/workflow-installation.rst

```diff
@@ -3,34 +3,82 @@ Install and run the CSET workflow
 for usage. For the Met Office installation this involves setting an environment
 variable before running cylc with the following commands:
 
-.. code-block::bash
+.. code-block:: bash
 
    export CYLC_VERSION=8
    cylc --version  # Check version starts in 8
 
+The workflow lives within the ``cset-workflow`` directory, and all the following
+commands assume you are within it.
+
+.. code-block:: bash
+
+  # From within your CSET repository, change into the workflow directory.
+  cd cset-workflow
+
+First up, if are at a specific site, you will want to install the site specific
+configuration. This is done by running the ``install_restricted_files.sh``
+script, and pressing enter a couple times to accept the default options.
+
+.. code-block:: bash
+
+  ./install_restricted_files.sh
+  # Then accept the defaults.
+
 You will then need to edit the configuration to set up your job. Start by making
 a copy of the ``rose-suite.conf.example`` file called ``rose-suite.conf``. This
 needs further amendments, which are done using `rose edit`_, a GUI configuration
 tool. Within rose edit go through the sections under "suite conf" and fill in
-the config. Help can be viewed by clicking on the name of a variable. Once you
-have configured CSET, you can save and close rose edit.
+the config.
+
+.. code-block:: bash
+
+    # Copy the example configuration.
+    cp rose-suite.conf.example rose-suite.conf
+    # Edit rose-suite.conf with the rose edit GUI.
+    rose edit &
+
+.. image:: rose-edit.png
+    :alt: rose edit GUI, showing the environment setup options.
+
+* The Data and Cycling section contains settings for the input data, and the
+  workflow cycling controls.
+
+* The Diagnostic section contains settings for enabling the various diagnostics
+  included with CSET. Some general diagnostics will take from a list of variable
+  names or STASH codes, and operate on all of them.
+
+* The Environment section contains settings for controlling how CSET integrates
+  with your compute environment. This includes things like activating modules,
+  and choosing a site profile.
+
+* The General section contains whole-workflow configuration for things like
+  logging and housekeeping.
+
+Help for each variable can be viewed by clicking on the variable's name. Once
+you have configured CSET you can save and close rose edit.
+
+To run the workflow, use ``cylc vip .``. This submits the workflow to a cylc
+scheduler. You can view the job's progress with the cylc GUI, accessible with
+the command ``cylc gui``.
+
+.. code-block:: bash
 
-Opening a command line inside the root of the CSET folder, you navigate to the
-``cset-workflow`` directory and then you can run it with the command ``cylc vip
-.``, which will cause cylc to submit the job to your site's batch processing
-system, and run it. You can view the job's progress with the cylc gui,
-accessible with the command ``cylc gui``.
+    # Run workflow
+    cylc vip .
+    # View workflow progress
+    cylc gui
```

Comment:
I guess the concern here would be portibility. While I personally run cylc gui under screen with those options, I do this because its a bit laggy on my virtual linux desktop, and that allows me to view it from windows. But for many people it is simpler to just do cylc gui and have it deal with opening a browser, etc.

On the other hand, many sites may have their linux environments being non-graphical, in which case having these options is the correct call.

I've opened an issue (#478) for this so we can deal with it seperately.